src/app/navigation/navigation.component.html

```diff
@@ -57,7 +57,7 @@
                   <a class="nav-link" href="" mdbWavesEffect>Register</a>
               </li>
               <li class="nav-item">
-                  <a class="nav-link" href="" mdbWavesEffect>Sign In</a>
+                  <a class="nav-link" href="signin" mdbWavesEffect>Sign In</a>
```

Comment:
The reason you are having some issues is because you are using the HTML href element.. Instead use the Angular routerLink element like on line 51.. should look like routerLink="signin" no href at all..